src/command_modules/azure-cli-acr/README.rst

```diff
@@ -84,23 +84,36 @@ Update a container registry
 
     Command
         az acr update: Update a container registry.
-
+    
     Arguments
-        --name -n      [Required]: Name of container registry.
-        --disable-admin          : Disable admin user.
-        --enable-admin           : Enable admin user.
-        --resource-group -g      : Name of resource group.
-        --storage-account-name -s: Name of an existing storage account.
-        --tags                   : Space separated tags in 'key[=value]' format. Use "" to clear
-                                 existing tags.
-
+        --name -n [Required]: Name of container registry.
+        --resource-group -g : Name of resource group.
+    
+    Generic Update Arguments
+        --add               : Add an object to a list of objects by specifying a path and key value
+                              pairs.  Example: --add property.listProperty <key=value, string or JSON
+                              string>.
+        --remove            : Remove a property or an element from a list.  Example: --remove
+                              property.list <indexToRemove> OR --remove propertyToRemove.
+        --set               : Update an object by specifying a property path and value to set.  Example:
+                              --set property1.property2=<value>.
+    
+    Global Arguments
```

Comment:
I think you have remove global arguments from your README elsewhere. They are global so no need to repeat them on every command :grin: